spec/features/publish_record_spec.rb

```diff
@@ -0,0 +1,39 @@
+describe "managing registers", :type => :feature do
+  before :each do
+    visit '/'
+    expect(page).to have_content 'Sign in'
+    fill_in 'Email', with: 'admin@gov.uk'
+    fill_in 'Password', with: 'password123'
+    click_button 'Continue'
+    expect(page).to have_content 'Signed in successfully.'
+end
+    
+    
+  it "publishes an update to the register" do
+    visit 'country/new'
+    expect(page).to have_content 'Country'
+    fill_in 'country', with: 'zz'
+    fill_in 'name', with: 'name'
+    fill_in 'official-name', with: 'official name'
+    fill_in 'start-date', with: '2014-05'
+    fill_in 'end-date', with: '2014-05'
+    click_button 'Continue'
+    
+    expect(page).to have_content 'Check the details of your new country'
+    click_button 'Submit'
+      
+    visit '/country#updates'
+    expect(page).to have_content 'zz'
+    review_items = page.all(:xpath, '//a[contains(text(), "Review")]')
```

Comment:
this is a really nasty selector to get around the fact there are two Review links on the page. I think it would be nicer to do something like: get `td` with text `zz`, get first sibling `a` but I couldn't work out a selector for that,

Reply:
Agreed. I'm in favour of adding a Class, ID, data attribute to the buttons to keep this cleaner